tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -167,6 +167,25 @@ Get User Notebook Pod Name
   ${notebook_pod_name}=   Set Variable  jupyterhub-nb-${safe_username}
   [Return]  ${notebook_pod_name}
 
+Wait Until User Server Is Clean
+    [Documentation]    Waits until the JL UI does not show any items (folders/files) in the user's server
+    [Arguments]    ${timeout}=30s
+    Wait Until Page Does Not Contain Element    xpath://li[contains(concat(' ',normalize-space(@class),' '),' jp-DirListing-item ')]    ${timeout}
+
+# Could replace Clean Up User Notebook and Delete Folder In User Notebook
+Exec Command In User Notebook
+    [Documentation]    Runs ${command} in the user's server and returns the result
+    [Arguments]    ${command}    ${admin_username}=${OCP_ADMIN_USER.USERNAME}    ${username}=${TEST_USER.USERNAME}
+    ${oc_whoami} =  Run   oc whoami
+    IF    '${oc_whoami}' == '${admin_username}'
+        ${notebook_pod_name} =   Get User Notebook Pod Name  ${username}
+        Search Pods    ${notebook_pod_name}    namespace=rhods-notebooks
+        ${output} =  Run    oc exec ${notebook_pod_name} -n rhods-notebooks -- ${command}
```

Comment:
It could be replaced by `Run Command In Container` in Pods.robot

Reply:
I have tested it but it was giving me issues with the returned value from the command execution. Something that also looked strange was this log message:
`Running command 'oc exec jupyterhub-nb-ldap-2duser4 -n rhods-notebooks -- ls 2>&1'.`
I am unsure if `2>&1` was just an artifact of the logs or was actually being passed as part of the command. Either way I won't have time to look into it, so I'd prefer to merge it like this since it's known to be working and we can investigate more when there's time.

Reply:
(looks like it combines stderr and stdout outputs, I'm guessing it's automatically appended by either RF or oc; https://stackoverflow.com/questions/818255/in-the-shell-what-does-21-mean)

Reply:
nevermind, got it working. 
@jgarciao could you update the `Clean Up User Notebook` and `Delete Folder In User Notebook` to use `Run Command In Container` or update the TCs that use them to instead use `Run Command In Container` directly, so that we can unify everything under one keyword?

Reply:
I will!